components/test_raftstore/src/server.rs

```diff
@@ -138,14 +139,14 @@ impl Simulator for ServerCluster {
             &tikv::config::StorageReadPoolConfig::default_for_test(),
             raft_engine.clone(),
         );
-        let store = create_raft_storage(
-            RaftKv::new(sim_router.clone()),
-            &cfg.storage,
-            storage_read_pool,
-            None,
-            None,
-            None,
-        )?;
+
+        let engine = RaftKv::new(sim_router.clone());
+
+        let mut gc_worker =
+            GCWorker::new(engine.clone(), None, None, cfg.storage.gc_ratio_threshold);
+        gc_worker.start().unwrap();
```

Comment:
Is it unnecessary? @MyonKeminta

Reply:
Of cource we can make it RAII, just like how `storage` does. I think it's a good idea.